scripts/deploy.ts

```diff
@@ -73,6 +77,17 @@ interface BuildObj extends BuildConfigFile {
   ipaFile: string // Also APK
 }
 
+const asLatestTestFile = asObject({
```

Comment:
Use `asJSON` and `asCodec`?

Reply:
Hmm, upon further review, the cleaner is never really used. Why not just a static type then?

Reply:
It was going to be used and for now the data doesn't need to be read but only written. Will change